recipes/apistar/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "apistar" %}
+{% set version = "0.5.4" %}
+{% set sha256 = "98a1b4650504139c76825307e1437830887eda1ae4c3774afb3c59f813af8524" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - aiofiles
+    - jinja2
+    - python
+    - requests
+    - werkzeug
+    - whitenoise
+
+test:
+  imports:
+    - apistar
+
+about:
+  home: https://github.com/encode/apistar
+  license: BSD 3-Clause
+  license_family: BSD
```

Comment:
It looks like the `license_file` is not included in the source distribution. Please manually package it:

https://conda-forge.org/docs/meta.html#packaging-the-license-manually

Also could you raise an issue with the developers to include the `license_file` in the source distribution and copy the url for the issue into the recipe as a comment. This acts as a reminder when you go to update the recipe in the future to check whether the `license_file` might be included and will no longer need to be included in the recipe since it will come in via the source.

Reply:
Hm, it's not clear to me how to resolve this change request automatically. Should be done with https://github.com/conda-forge/staged-recipes/pull/5707/commits/18fc35716c907a869cb3cc74b1273a3986ed0ffd